collector/disk_usage.go

```diff
@@ -0,0 +1,109 @@
+package collector
+
+import (
+	"context"
+
+	pgx "github.com/jackc/pgx/v4"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Queries taken from https://wiki.postgresql.org/wiki/Disk_Usage
+const (
+	relationUsageQuery = `
```

Comment:
as we discuss:

```
SELECT schemaname
, indexrelname
, pg_relation_size(indexrelid)::float
FROM pg_stat_user_indexes ORDER BY indexrelname DESC;
```

This will be focus only in the index sizes and avoids sequence, toast tables and other things.